### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by CHEF-KOCH**
* Simplify compile instructions 
* Missing projects added (SimpleNoSQL & Material-Dialogs)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +164,11 @@ This project also uses many other open source libraries such as:
         <td>Apache License 2.0</td>
         <td>https://github.com/Stericson/RootTools</td>
     </tr>	
+    <tr>
```
Comment (ukanth) [inline README.md:167]: I no longer use SimpleNoSQL. Can you remove it and send PR again ?

**Comment by CHEF-KOCH** [comment]
I would but somehow I lost my wiki + commit rights. So I can't edit my pull.

**Comment by ukanth** [comment]
Hey, sorry @CHEF-KOCH . I was playing around with some github apps and that messed up the access part. I removed all of those access for wiki. Sorry for that again !

**Comment by CHEF-KOCH** [comment]
I need to close and re-bump it with other changes, somehow I deleted my working repo so I can't edit this.

**Pull request closed without merging**